server/src/static/assets/js/filter.js

```diff
@@ -1,19 +1,15 @@
-function runFilter() {
-  var input, filter, table, tr, td, i, txtValue;
-  input = document.getElementById("inputFilter");
-  filter = input.value.toUpperCase();
-  table = document.getElementById("filterTable");
-  tr = table.getElementsByTagName("tr");
+document.addEventListener('DOMContentLoaded', function() {
+    const searchBox = document.querySelector('.p-search-box__input');
+    const tableRows = Array.from(document.querySelectorAll('.p-table--mobile-card tbody tr'));
```

Comment:
While these selectors work, I'm worried that any changes to the style used would break this again. Wdyt if we set an id or class for searchBox (e.g. `search-box`) and a class for rows (e.g. `searchable-row`) and select on these instead? This way future modifications to style of these wont break functionality.

Reply:
Good idea, however I think it might be unnecessary for the existing ".p-search-box__input" ones since that's already clearly intended as a search box? See what you think, but I did add a "searchable-row" class and restricted the js search bits to only look at those. See what you think.

Reply:
Yeah, actually it makes sense that search box is a search box (unless we end up with multiple search boxes in the future) so it's probably an overkill to create a class for it. I like the addition of the `searchable-row`. I was thinking of selecting on `searchable-row` only though. That way if any of the parents `.p-table--mobile-card tbody tr` change the search wont break. But that's optional.